guides/micronaut-google-cloud-platform-cloud-run/micronaut-google-cloud-platform-cloud-run.adoc

```diff
@@ -183,13 +183,15 @@ Service [micronautguide] revision [micronautguide-00002-fat] has been deployed
  https://micronautguide-li3tercjmq-ue.a.run.app
 ----
 
-
 == Running the Application
 
 [source, bash]
 ----
 curl -i https://micronautguide-li3tercjmq-ue.a.run.app/hello
-HTTP/2 200 
+----
+
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks